pages/apis/rest_api/analytics/flaky_tests.md.erb

```diff
@@ -0,0 +1,32 @@
+# Flaky tests API
+
+{:toc}
+
+This feature is currently in beta, and not available to all organizations. If you would like early access, please TBC.
+
+## List flaky tests
+
+Returns a [paginated list](<%= paginated_resource_docs_url %>) of the user's flaky tests.
```

Comment:
This looks like it's tied to a test suite rather than a user? If so, perhaps:

```suggestion
Returns a [paginated list](<%= paginated_resource_docs_url %>) of the flaky tests detected in a test suite.
```